main.tf

```diff
@@ -64,6 +63,16 @@ resource "aws_s3_bucket" "kinesis_firehose_s3_bucket" {
   tags = var.tags
 }
 
+resource "aws_s3_bucket_public_access_block" "kinesis_firehose_s3_bucket" {
+  count  = "${var.s3_bucket_block_public_access_enabled}"
+  bucket = "${aws_s3_bucket.kinesis_firehose_s3_bucket.id}"
```

Comment:
Did TF13 not complain about the syntax in this resource where the values for count and bucket are wrapped in the old style TF11 of `"${ <resource> }"?  Why not just `aws_s3_bucket.kinesis_firehose_s3_bucket.id` here?

Reply:
You know it probably did, but we use terragrunt here and sometimes those warnings get lost in the wrapper messages.  I will update this